heka/_service.sls

```diff
@@ -191,6 +191,7 @@ heka_{{ service_name }}_grain:
   - group: heka
   - defaults:
     service_name: {{ service_name }}
+    poolsize: {{ server.poolsize|default(100) }}
```

Comment:
yes I thought about it but the map.jinja is not used in this sls and second all services have their own default in metadata/service.
I would even remove the "|default(100)".

anyway I propose a patch